iea_18/openems_dev_env/README.md

```diff
@@ -0,0 +1,18 @@
+# IEA Task 18 openEMS development environment
+
+This docker compose setup provides:
+
+ * [openems-ui](http://openems-ui.localhost/)
+ * [InfluxDB](http://influxdb.localhost/)
+ * [Grafana](http://grafana.localhost/)
+
+To start, run ```docker compose up -d```. To stop it, run ```docker compose down```. To also remove all volumes and thus all data in influxdb, run ```docker compose down -v```.
+
+With openEMS running, use [Apache Felix](http://localhost:8080/system/console/configMgr) to configure the "Timedata InfluxDB" plugin:
```

Comment:
it asks me for a username and password here

Reply:
@Bachibouzouk The default: admin:admin